keps/sig-instrumentation/3466-kubernetes-component-health-slis/README.md

```diff
@@ -250,13 +250,13 @@ without any issues.
 
 ##### Integration tests
 
-- [ ] ensure existence of healthcheck endpoint (beta requirement)
+- [X] ensure existence of healthcheck endpoint (beta requirement)
```

Comment:
IIUC, these tests don't exist now. So can you please unmark it?

Reply:
https://github.com/kubernetes/kubernetes/pull/114997

Reply:
So they don't exist (as long as it's not merged, it's doesn't really exist :) )

Reply:
Okay I'll leave them unchecked until later.